apps-rendering/src/components/byline.tsx

```diff
@@ -87,12 +87,17 @@ const labsStyles = css`
     `}
 `;
 
-const liveblogStyles = (kicker: string): SerializedStyles => css`
+const liveblogStyles = (link: string): SerializedStyles => css`
 	${headline.xxxsmall({ lineHeight: 'regular', fontStyle: 'italic' })}
 	color: ${palette.neutral[93]};
-
 	${from.desktop} {
-		color: ${kicker};
+		color: ${link};
+	}
+	a {
+		color: ${palette.neutral[93]};
+		${from.desktop} {
+			color: ${link};
+		}
```

Comment:
Great point @JamieB-gu , moved the `a` styling to the `liveblogAnchorStyles`.  Also fixed the dark mode (Although in general I want to focus on dark mode in a separate PR)
Just something to note here is that I had to apply the colors had both on `address` and `anchor` because the colors on address were affecting the content between the anchors, but not the anchors themselves. The anchors were getting `user agent stylesheet` colors. 
![image](https://user-images.githubusercontent.com/15894063/140294400-c6424b6e-bb60-4e5b-9ebe-ed6b297001b4.png)
![image](https://user-images.githubusercontent.com/15894063/140294778-e352b9cc-227f-46d0-8558-62d266246bbb.png)

Also by applying colors obly on anchor was not enough because the content between the anchors needed styling too. 
![image](https://user-images.githubusercontent.com/15894063/140294651-af829b99-e9f5-4c2c-9939-db234f7e35b6.png)